Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.nuspec

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>EdFi.Ods.SandboxAdmin.Web</id>
+    <id>EdFi.Ods.SandboxAdmin</id>
     <version>0.0.0</version>
     <authors>Ed-Fi Alliance</authors>
     <owners>Ed-Fi Alliance</owners>
     <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>EdFi.Ods.SandboxAdmin.Web</description>
+    <description>EdFi.Ods.SandboxAdmin</description>
     <releaseNotes></releaseNotes>
     <license type="file">LICENSE.txt</license>
   </metadata>
   <files>
-    <file src="../../LICENSE.txt" />
-    <file src="../../NOTICES.md" />
+    <file src="../../../LICENSE.txt" />
+    <file src="../../../NOTICES.md" />
     <file src="**" />
```

Comment:
we should consider excluding appsettings.development.json and appsettings.user.json

Reply:
Files excluded